components/organisms/o-shipping.vue

```diff
@@ -47,15 +47,12 @@
         :error-message="$t('Field is required')"
         @blur="$v.shipping.streetAddress.$touch()"
       />
+      <!-- https://github.com/building-21/pos-erp/issues/938 -->
```

Comment:
Since we are doing this at the adapter level, please remove this comment from here ( I originally thought you would do the mapping in VSF) and add it here: 
https://github.com/building-21/pos-erp/pull/931/files/85a517ccd2becd1991d4a3e2d66f2fbfbdfad641#r625486188

Reply:
sure.